File: apps/problems/itenary/full-boilerplate/function.cpp


        ###USER_CODE_HERE

         int main() {
              vector<vector<string>> tickets1 = {{"MUC", "LHR"}, {"JFK", "MUC"}, {"SFO", "SJC"}, {"LHR", "SFO"}};
vector<string> output1 = findItinerary(tickets1);
cout << "[";
          for (int i = 0; i < output1.size(); i++) {
              cout << '\"' << output1[i] << '\"';
              if (i < output1.size() - 1) {
                  cout << ",";
              }
          }
          cout << "]" << endl;
  vector<vector<string>> tickets2 = {{"JFK", "SFO"}, {"JFK", "ATL"}, {"SFO", "ATL"}, {"ATL", "JFK"}, {"ATL", "SFO"}};
vector<string> output2 = findItinerary(tickets2);
cout << "[";
          for (int i = 0; i < output2.size(); i++) {
              cout << '\"' << output2[i] << '\"';
              if (i < output2.size() - 1) {
                  cout << ",";
              }
          }
          cout << "]" << endl;
        return 0;
    }
        
       

File: apps/problems/itenary/full-boilerplate/function.java

            public class Main {
                public static void main(String[] args) {
                    List<List<String>> tickets1 = Arrays.asList(
          Arrays.asList("MUC", "LHR"),
  Arrays.asList("JFK", "MUC"),
  Arrays.asList("SFO", "SJC"),
  Arrays.asList("LHR", "SFO")
        );
List<String> output1 = findItinerary(tickets1);

         System.out.print("[");
          for (int i = 0; i < output1.size(); i++) {
               System.out.print("\"" + output1.get(i) + "\"");
              if (i < output1.size() - 1) {
                  System.out.print(", ");
              }
          }
          System.out.println("]");
 List<List<String>> tickets2 = Arrays.asList(
          Arrays.asList("JFK", "SFO"),
  Arrays.asList("JFK", "ATL"),
  Arrays.asList("SFO", "ATL"),
  Arrays.asList("ATL", "JFK"),
  Arrays.asList("ATL", "SFO")
        );
List<String> output2 = findItinerary(tickets2);

         System.out.print("[");
          for (int i = 0; i < output2.size(); i++) {
               System.out.print("\"" + output2.get(i) + "\"");
              if (i < output2.size() - 1) {
                  System.out.print(", ");
              }
          }
          System.out.println("]");


                }
                ###USER_CODE_HERE
            }
        